README.md

```diff
@@ -123,53 +120,59 @@ ez_recommendation:
         script_url: 'cdn.yoochoose.net/yct.js'
 ```
 
-**IMPORTANT**
-Changing any of the parameters above without a valid reason will break all calls to YOOCHOOSE.
+Changing any of the parameters above without a valid reason will break all calls to YOOCHOOSE. It can be useful to test the API by mocking the service, or if you have a hosted version of YOOCHOOSE Recommendation service.
+
+## Step 3: Clear `prod` cache
 
-*Possible use cases when parameters can be changed:*
-- testing the API by mocking the service
-- integrating with a hosted version of YOOCHOOSE Recommendation service
+While Symfony `dev` environment keeps track of changes to yml files, `prod` does not, so to make sure Symfony reads the new config we clear cache:
+```
+php app/console --env=prod clear:cache
```

Comment:
be aware the command is `php ezpublish/console --env=prod clear:cache` on eZ Publish 5.x.